### ISSUE TITLE
I propose testing the react-testing test.

### ISSUE THREAD

**Issue by r-saba** [TheGreatMarkus/react-testing-test#1]
This will ensure that we have thoroughly tested the test allowing us to be confident in the test.

**Comment by TheGreatMarkus** [comment]
Good point, if we don't test the test of the test, who gonna make sure that the test of the test is behaving according to specs?
